utils/env_preferences/src/parse/aliases.rs

```diff
@@ -4,59 +4,59 @@
 
 //! Platform-specific conversion from locale strings to BCP-47 identifiers.
 
-/// Find a BCP-47 language/region from a list of POSIX locale aliases.
+/// Find a BCP-47 language/region from a list of POSIX 'locale name' aliases.
 ///
-/// Some of these conversions are approximate and not exact (e.g. "C"->"und").
 /// This is based on GNU libc's `intl/locale.alias` file, with some manual processing
-/// to remove duplicates. The original file is available at:
+/// to remove duplicates. The notable exception is that the default `C`/`POSIX` locales
+/// map to `en-US`. The original file is available at:
```

Comment:
```suggestion
/// map to `en-US-posix`. The original file is available at:
```

Reply:
Resolved as the alias code has been removed